libs/cereal/Makefile

```diff
@@ -26,7 +26,9 @@ CMAKE_OPTIONS += -DCMAKE_CXX_FLAGS=-latomic -DWITH_WERROR=OFF -DSKIP_PORTABILITY
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
 
-define Package/cereal-dev
+
+define Package/cereal
+  HIDDEN:=1
```

Comment:
I'd rather use BUILDONLY. It accomplishes the same thing I think.

Reply:
OK.